### PR TITLE
[Feature] Add netlify ci

### PR DESCRIPTION
**Pull request by levin-du**
Add netlify ci support

### CODE REVIEW

**Comment by sibiraj-s** [comment]
Thanks @levin-du .

File: README.md [hunk 1]
```diff
@@ -19,47 +19,48 @@ npm install ci-info --save
 ## Usage
 
 ```js
-var ci = require('ci-info')
+var ci = require('ci-info');
```
Comment (sibiraj-s) [inline README.md]: Can you revert the changes that are not relevant to this PR.

**Comment by sibiraj-s** [comment]
@levin-du Can you post a link for the docs where `NETLIFY` environment variable is documented. I couldn't find one.

**Comment by levin-du** [comment]
@sibiraj-s 
- `PULL_REQUEST`:https://www.netlify.com/docs/continuous-deployment/#build-environment-variables
- `NETLIFY_BUILD_BASE `: https://github.com/netlify/build-image/blob/master/run-build.sh#L5 , also can be found in `env` output.

File: vendors.json [hunk 2]
```diff
@@ -148,5 +148,11 @@
     "constant": "TRAVIS",
     "env": "TRAVIS",
     "pr": { "env": "TRAVIS_PULL_REQUEST", "ne": "false" }
+  },
+  {
+    "name": "Netlify CI",
+    "constant": "NETLIFY",
+    "env": "NETLIFY_BUILD_BASE",
+    "pr": { "PULL_REQUEST": "true" }
```
Comment (sibiraj-s) [inline vendors.json]: The could be

```suggestion
    "pr": { "env": "PULL_REQUEST" }
```

Just to maintain consistency among env variables

Comment (levin-du) [inline vendors.json]: @sibiraj-s  We can not do that as `PULL_REQUEST` will be `false` for not pr .

Comment (levin-du) [reply vendors.json]: @sibiraj-s We can not do that as PULL_REQUEST will be false for not pr .

**Comment by levin-du** [comment]
@sibiraj-s It seems you can merge this pr now.  : )

**Comment by watson** [comment]
Support for Netlify has been released in v2.0.0
